src/pages/VaccineModulePage.js

```diff
@@ -0,0 +1,281 @@
+/* eslint-disable react/forbid-prop-types, global-require */
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2019
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Image, StyleSheet, View, Text, TouchableOpacity } from 'react-native';
+import { Button } from 'react-native-ui-components';
+import Icon from 'react-native-vector-icons/FontAwesome';
+import { VaccineChart } from '../widgets';
+import globalStyles, {
+  SHADOW_BORDER,
+  APP_FONT_FAMILY,
+  SUSSOL_ORANGE,
+  GREY,
+  HAZARD_RED,
+} from '../globalStyles';
+
+const CHEVRON_ICON_STYLE = { size: 18, color: SUSSOL_ORANGE };
+const BREACH_ICON_STYLE = { size: 25, color: HAZARD_RED };
+
+const MILLISECONDS_IN_DAY = 24 * 60 * 60 * 10000;
+const MAX_LOOKBACK_MILLISECONDS = 30 * MILLISECONDS_IN_DAY;
+
+// TODO navigation for menu buttons
+// TODO Localise all strings
+
+export class VaccineModulePage extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { fridges: [], selectedFridgeId: null };
+  }
+
+  componentWillMount() {
+    const { database } = this.props;
+    const fridges = database.objects('Location').filter(({ isFridge }) => isFridge);
```

Comment:
Not a requested change: I think fridges is better off as a constant. It will not change and will only increase the time spent for state comparisons in updating. 
Also hasFridges could be a value/constant also

[ I really don't like things not in state, but as per other pages it seems right to keep things that don't change and are not primitive data types outside of state ]

Reply:
https://github.com/openmsupply/mobile/pull/875/commits/dd276662e2f033e57a90e6384a5ce6d525e39054
https://github.com/openmsupply/mobile/pull/875/commits/ded021597ae6732a825080020d1ab17e22f3e4e8